alerting/src/test/kotlin/org/opensearch/alerting/resthandler/MonitorRestApiIT.kt

```diff
@@ -569,6 +569,129 @@ class MonitorRestApiIT : AlertingRestTestCase() {
         assertFalse("Alert in state ${activeAlert.state} found in failed list", failedResponseList.contains(activeAlert.id))
     }
 
+    fun `test acknowledging more than 10 alerts at once`() {
+        // GIVEN
+        putAlertMappings() // Required as we do not have a create alert API.
+        val monitor = createRandomMonitor(refresh = true)
+        val alertsToAcknowledge = arrayOf(
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE)),
+            createAlert(randomAlert(monitor).copy(state = Alert.State.ACTIVE))
+        )
+
+        // WHEN
+        val response = acknowledgeAlerts(monitor, *alertsToAcknowledge)
+
+        // THEN
+        val responseMap = response.asMap()
+        val expectedAcknowledgedCount = alertsToAcknowledge.size
+
+        val acknowledgedAlerts = responseMap["success"] as List<String>
+        assertTrue("Expected $expectedAcknowledgedCount alerts to be acknowledged successfully.", acknowledgedAlerts.size == expectedAcknowledgedCount)
+
+        val acknowledgedAlertsList = acknowledgedAlerts.toString()
+        alertsToAcknowledge.forEach { alert -> assertTrue("Alert with ID ${alert.id} not found in failed list.", acknowledgedAlertsList.contains(alert.id)) }
+
+        val failedResponse = responseMap["failed"] as List<String>
+        assertTrue("Expected 0 alerts to fail acknowledgment.", failedResponse.isEmpty())
+
+        val failedResponseList = failedResponse.toString()
```

Comment:
Since we're asserting above that there are no alerts in the `failed` response, I think we can remove this check since the test would have failed prior to reaching here in the case where there is something to check.